index.js

```diff
@@ -107,9 +116,10 @@ Object.defineProperties(
 				return new Ec2({ region: this.serverless.service.provider.region });
 			}),
 			functionNames: d(function () {
+				const warmupFunction = this.serverless.service.custom.warmup ? [this.serverless.service.custom.warmup.name] : [];
```

Comment:
`custom` may not be set, and actually CI fails on that.

Additional question do `custom.warmup` comes from some plugin?

I think for such custom extra lambdas (from whatever plugins) might be good to propose special `option`, from which we may read extra names (?)

Reply:
That's a better idea let us do that instead 👍